packages/cashier/src/stores/withdraw-store.ts

```diff
@@ -231,7 +231,7 @@ export default class WithdrawStore {
 
                 client.setVerificationCode('', container);
             }
-        } else if (is_crypto) {
```

Comment:
Hey @jim-deriv 
I have a suggestion.
With `XRP` currency introduction, BE also introduced new field in website_status.currencies_config[currency] which is platform. Platform has two properties: cashier: `'doughflow'` | `'crypto' `and ramp - onramp providers.
In order to proper detect cashier provider and currency type, now in cashier package we are trying to use this predicate: website_status?.currencies_config[currency].platform.cashier.includes('crypto'). 
So my suggestion is to add into cashier `general store` this part:
```
   get is_crypto_provider(): boolean {
        const { currency, website_status } = this.root_store.client;
        //@ts-expect-error needs type from backend
        return !!website_status?.currencies_config[currency].platform.cashier.includes('crypto');
    }
```
 and use it in withdrawal store instead of is_crypto in `onMountWithdraw` method.
```
  const is_crypto_provider = modules.cashier?.general_store.is_crypto_provider;
 } else if (is_crypto_provider) {
```
So in future if we will add new currency there is no need to add more conditions like: `client.currency !== 'XRP'`.